src/main/scala/com/johnsnowlabs/reader/WordReader.scala

```diff
@@ -31,6 +31,49 @@ import java.io.{ByteArrayInputStream, IOException}
+  * +----------------------------------------------------------------------------------------------------------------------------------------------------+
+  * |[{Table, Header Col 1, {}}, {Table, Header Col 2, {}}, {Table, Lorem ipsum, {}}, {Table, A Link example, {}}, {NarrativeText, Dolor sit amet, {}}]  |
+  * +----------------------------------------------------------------------------------------------------------------------------------------------------+
+  *
+  * wordDf.printSchema()
+  * root
+  *  |-- path: string (nullable = true)
+  *  |-- doc: array (nullable = true)
+  *  |    |-- element: struct (containsNull = true)
+  *  |    |    |-- elementType: string (nullable = true)
+  *  |    |    |-- content: string (nullable = true)
+  *  |    |    |-- metadata: map (nullable = true)
+  *  |    |    |    |-- key: string
+  *  |    |    |    |-- value: string (valueContainsNull = true)
+  *
+  * }}}
+  */
+
```

Comment:
It might be helpful to include a reference link to the WordReader demo notebook that has examples with all available parameters: `examples/python/reader/SparkNLP_Word_Reader_Demo.ipynb`